components/dropdown/dropdown-menu.js

```diff
@@ -91,8 +91,15 @@ class DropdownMenu extends DropdownContentMixin(LitElement) {
 	}
 
 	__getMenuElement() {
-		return this.shadowRoot.querySelector('.d2l-dropdown-content-container > slot')
-			.assignedNodes()
+		let slot = this.shadowRoot.querySelector('.d2l-dropdown-content-container > slot');
```

Comment:
I think I've seen this before actually... what if you try putting a CSS class name on the slot and then change this selector to be `.d2l-dropdown-content-slot`, does it still need the Edge hack then?

Reply:
Yeah there is some weirdness here.  As Dave mentioned, I think we should be able to get this working without two different selectors.